ui/packages/catalog-ui-search/src/main/webapp/component/dropdown/search-form-interactions/dropdown.search-form-interactions.view.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+    'marionette',
+    'underscore',
+    'jquery',
+    '../dropdown.view',
+    './dropdown.search-form-interactions.hbs',
+    'component/search-form-interactions/search-form-interactions.view'
+], function (Marionette, _, $, DropdownView, template, ComponentView) {
+
+    return DropdownView.extend({
+        template: template,
+        className: 'is-searchFormInteractions',
```

Comment:
nit: `dropdown.is-search-form-interactions` for consistency

Reply:
Less file was not needed because search-form.less is dictating look and feel. Files was removed